fetch.js

```diff
@@ -177,6 +177,8 @@
         this._bodyText = body.toString()
       } else if (!body) {
         this._bodyText = ''
+      } else if (body.getReader) {
+        this.body = body
```

Comment:
It's also suppose to solve abortable request by calling `res.body.cancel()` So literally every thing you pass into `new Response` should be transformed to ReadableStream IMO.

But I don't want to rush into things and change the hole polyfill